metadb/scrapers/musicbrainz.py

```diff
@@ -0,0 +1,43 @@
+import getpass as gt
+import sys
+import musicbrainzngs as mb
+
+#username = raw_input('Enter MusicBrainz Username: ')
+#password = gt.getpass('Password for {}: '.format(username))
+#mb.auth(username,password)
+
+mb.set_useragent(
+    "python-metadb-musicbrainz-scrape",
+    "0.1",
+    "https://github.com/MTG/metadb")
+
+
+
+def scrape(mbid):
+    try:
+        result = mb.get_recording_by_id(mbid, includes=["artists", "releases", "tags"])
+	
+	recordingName = result["recording"]["title"] #recording title
```

Comment:
be careful about indentation here. It looks like you're mixing tabs and spaces in this file. It's common in python code to indent with 4 spaces